packages/node-resolve/README.md

```diff
@@ -13,7 +13,7 @@
 
 ## Requirements
 
-This plugin requires an [LTS](https://github.com/nodejs/Release) Node version (v8.0.0+) and Rollup v1.20.0+.
+This plugin requires an [LTS](https://github.com/nodejs/Release) Node version (v10.0.0+) and Rollup 2.78.0+.
```

Comment:
I know this contradicts what @lukastaegert had said, but we only support actual LTS versions of Node. https://nodejs.org/en/about/releases/ Right now, that's v14+

I'll fast-follow this PR with a package.json update.